### DISCUSSION TITLE
Add DevTools Support for Frame apps that were built without Frog

### DISCUSSION THREAD

**Discussion by PirosB3** [wevm/frog-archived#57] [Ideas]
I've really enjoyed prototyping using Frog's Frame Devtools - and I consider the UI incredibly intuitive. We have invested in our own Frames framework, the cost of switching to Frog is too high. However, I would still love to integrate Frog within my workflow as a testing tool.

### Objective
Allow Frame Devtools (`frog dev`) to support debugging applications not developed with Frog. This initiative aims to broaden the use cases for Frog, which could not be used on any Frames app across a diverse range of frameworks.

### Integration points and requirements

- Can start `frog dev` outside of a project or launch with an external URL.
- Can navigate to frame URLs on localhost and on remote servers
- Should display existing stats when still applicable (request tab, meta tags, latency, image size)
- Should be able to handle `post_redirect` buttons, allowing user to open the redirect URL in a separate tab
- Should detect and display irregularities with the OG responses, if they do not comply with Farcaster's spec.

Let me know what you think, and thanks again for building Frog

**Comment by Complexlity** [comment]
Hey man. I asked the same thing here:
https://github.com/wevm/frog/discussions/44

The frog debugger has some context embedded so that'll need to be stripped before it can be used in a non-frog frame but according to @tmm . It's in the works

**Comment by tmm** [comment]
Added support for this in `frog@0.7`

https://warpcast.com/awkweb/0x77ce28e1
